config/doctrine/Attendance/Attendance.orm.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
+                          https://www.doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+
+    <entity name="App\Entity\Attendance\Attendance" table="attendances" repository-class="App\Repository\Attendance\AttendanceRepository">
+        <field name="missionId" type="string" length="255"/>
+        <field name="playerId" type="bigint"/>
+
+        <indexes>
+            <!-- This is needed because child index mappings override parent index mappings -->
+            <index columns="created_at"/>
+
+            <index columns="mission_id"/>
+            <index columns="player_id"/>
```

Comment:
Combo of `mission_id` + `player_id` should be unique, one player can not attend the same mission twice.